internal/syncd/integrations/github/listener_test.go

```diff
@@ -0,0 +1,97 @@
+package github
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/proto/hapi/chart"
+)
+
+type mockDownloader struct {
+	mock.Mock
+}
+
+func (m *mockDownloader) BufferDirectory(ctx context.Context, repo, path, ref string) ([]*chartutil.BufferedFile, error) {
+	args := m.Called(ctx, repo, path, ref)
+	var ret0 []*chartutil.BufferedFile
+	if args0 := args.Get(0); args0 != nil {
+		ret0 = args0.([]*chartutil.BufferedFile)
+	}
+
+	return ret0, args.Error(1)
+}
+
+type mockReconciler struct {
+	mock.Mock
+}
+
+func (m *mockReconciler) Reconcile(ctx context.Context, chart *chart.Chart) error {
+	args := m.Called(ctx, chart)
```

Comment:
`m.Called(ctx, chart)` could be inlined into the return statement like so ` return m.Called(ctx, chart).Error(0)`.